app/assets/stylesheets/govuk_publishing_components/components/_related-navigation.scss

```diff
@@ -42,10 +42,10 @@
 .gem-c-related-navigation__link {
   list-style-type: none;
   margin-top: govuk-spacing(3);
-  @include govuk-font($size: 16, $weight: regular, $line-height: 1.5);
+  @include govuk-font($size: 16, $weight: regular, $line-height: 1.45);
 
   @include govuk-media-query($from: tablet) {
-    margin-top: 5px;
+    line-height: 1.28;
```

Comment:
Just wondering about this... the line height on mobile is `1.45` and the overlap/underlap problem doesn't occur. I haven't tested it cross browser, but I can increase it to `1.42` (on desktop, 1.49 on mobile) before the black underline of the line above becomes visible.

Might it be worth making the line height as large as possible? I'm concerned about readability - I can read it fine, but others may find the lines too close together.

Reply:
Hmm, let me test those numbers in IE 11 and see.

Reply:
Okaydokey, here's what I found:

| Browser  | Maximum line height for large screens to avoid overlap | Maximum line height for smaller screens to avoid overlap |
| - | - | - |
| Chrome | 1.37 | 1.49 |
| Firefox | 1.45 | 1.46 |
| Safari | 1.37 | 1.49 |
| IE 11 | 1.41 | 1.28 |
| Edge | 1.34 | 1.36 |

Edge 18 is doing some really funky things, so it's difficult to tell what's overlap and what's glitch:

![Screen Shot 2019-08-13 at 17 05 28](https://user-images.githubusercontent.com/1732331/62957390-97e7e780-bdec-11e9-8c5e-67e8e4a555a6.png)

And IE is a pain to run inspect element in using BrowserStack, so I'll use my laptop at home to check both Edge and IE 11 this evening and report back tomorrow.

Reply:
Edge is a known issue and cannot be fixed we have decided to wait until Edge starts using Chromium since it's box-shadow rendering cannot be fixed by us. 😞

Reply:
I've checked on IE 11 and Edge 17 at home and the line heights look correct. 

Thanks for the info about Edge 18 @nickcolley - shame box-shadow doesn't work, but at least it doesn't make links unusable.